test.py

```diff
@@ -52,6 +52,50 @@ async def test_start_failed():
     assert service.running is False
 
 
+class Root(Simple):
+    def __init__(self, deps):
+        super(Root, self).__init__()
```

Comment:
In modern python it is enough to use just `super().__init__()`

Reply:
Fixed